operators/constellation-node-operator/go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.3.1
 	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.2.1
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v4 v4.1.0
-	github.com/edgelesssys/constellation/v2 v2.0.0
+	github.com/edgelesssys/constellation/v2 v2.5.2
 	github.com/edgelesssys/constellation/v2/3rdparty/node-maintenance-operator v0.0.0-00010101000000-000000000000
 	github.com/edgelesssys/constellation/v2/operators/constellation-node-operator/v2/api v0.0.0
```

Comment:
We should think about those too

Reply:
It feels like they should be upgraded in sync. Although i'm not sure if they are also on v2

Reply:
They didn't show up in the warnings so I didn't touch them. Let me quickly update them.

Reply:
Thinking about them again: They don't need updating. These modules are not part of the advisory. Right?

Reply:
They are versioned independent and are not v2